frontend/public/components/cluster-settings/ldap.jsx

```diff
@@ -471,7 +471,7 @@ const LDAPs = reduxForm({
         const populated = !!this.state.populated[stepName];
         const lastStep = i === Steps.length - 1;
         const step = <div key={`step-${stepName}`}>
-          { s.name && <h1 className="co-section-title ldap-group">{s.name}</h1> }
+          { s.name && <h2 className="co-section-heading ldap-group">{s.name}</h2> }
```

Comment:
I like the consistency, and wonder if there is (future) work that could be done to reduce duplication in some of these other headings as well.  Possibly taking the `<Heading>` component & letting it specify the `h` lvl as a prop, or other things... 

Just a passing thought.